_episodes/introduction.md

```diff
@@ -143,15 +143,19 @@ individual **l**ist **i**tem (`<li>` and `</li>`) tags.
 
 ## Static vs Dynamic Sites
 
-Sites consisting of pages whose content doesn't change
-after that initial information is received by the server
-are referred to as _static_ (for example, a website containing one's CV).
-Conversely, sites whose pages can be updated after loading,
-with information continuing to be passed between browser and server,
-are called _dynamic_ (for example, pages showing current date and time or shopping baskets, Twitter or Facebook pages,
-which might be updated between each two page refreshes).
-This lesson focuses on static sites and tools that can be used to create them,
-known as **Static Site Generators**.
+_Static_ pages are those whose contents are stored on a server in a state ready
+to be sent to any user who makes a request for that web page. When a request is
+made, the server only needs to send the information comprising that web page
+(such as HTML and CSS). Sites that do not change often, such as a website
+containing one’s CV, are often stored as static sites. Conversely, _dynamic_
```

Comment:
```suggestion
containing one’s CV, are often stored as static sites. 

Conversely, _dynamic_
```

Suggest to split the paragraph in two at this point.

Reply:
Splitting that paragraph looks good to me.